mmrotate/models/losses/gaussian_dist_loss.py

```diff
@@ -386,7 +386,9 @@ def forward(self,
             reduction_override if reduction_override else self.reduction)
         if (weight is not None) and (not torch.any(weight > 0)) and (
                 reduction != 'none'):
-            return (pred * weight).sum()
+            if pred.dim() == weight.dim() + 1:
```

Comment:
Why do we need to change this file?

Reply:
GDLoss got error when used with RTMDet-R.
Follow other losses in mmdet, handle the different input dim of weight.